editoast/src/views/rolling_stock/towed.rs

```diff
@@ -54,7 +54,8 @@ editoast_common::schemas! {
 #[editoast_derive::annotate_units]
 #[derive(Debug, Serialize, ToSchema)]
 #[cfg_attr(test, derive(serde::Deserialize, PartialEq))]
-struct TowedRollingStock {
+#[schema(as = TowedRollingStock)]
+struct TowedRollingStockResponse {
```

Comment:
Why do we need a different object for the response? It seems that it is only for the `version` field. Do we care enough?

To me, it's not really about a `Response`, but more about one object for `models`, and one object for `views`. And both could be named `TowedRollingStock` (that would also avoid the renaming above `#[schema(as = TowedRollingStock)]`.

Maybe `models::towed_rolling_stock::TowedRollingStock` can be reexposed in `models.rs` and therefore, use `models::TowedRollingStock` in the current file whenever needed, what do you think?

Reply:
It was like that before, I just renamed stuff, figured less changes would be better. Done in 096b47369